src/main/java/data_gateway/task/ObservableTaskManager.java

```diff
@@ -0,0 +1,10 @@
+package data_gateway.task;
+
+import data_gateway.ObservableRepository;
+
+/**
+ * Interface allowing implementors polymorphic access to
+ * {@link TodoListManager} and {@link ObservableRepository<TaskReader>}
+ */
+public interface ObservableTaskManager extends ObservableRepository<TaskReader>, TodoListManager {
```

Comment:
Name of the interface isn't standardized with the one for event (`ObservableEventRepository`)